app.rb

```diff
@@ -15,14 +15,12 @@ def list_books(_books)
     @books.each_with_index do |book, index|
       puts "#{index + 1}) Title: \"#{book.title}\", Author: #{book.author}"
     end
-    puts ''
   end
 
   def list_people(_people)
     @people.each do |person|
       puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
     end
-    puts ''
   end
 
   def create_person
```

Comment:
- [ ] In respect to `Single Responsibility Principle`, a class should have a single purpose and should be focused on doing that one thing well. Kindly take note the this methods do not follow `SRP `because, `all requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate input.`
## **Kindly implement this for **create_student, create_teacher, create_book, create_rentals and list_rentals** methods**